components/multiselect/multiselect_list.jsx

```diff
@@ -27,6 +31,7 @@ export default class MultiSelectList extends React.Component {
 
     componentDidMount() {
         document.addEventListener('keydown', this.handleArrowPress);
+        loadStatusesForProfilesList(this.props.options);
```

Comment:
We shouldn't be adding user-specific stuff to the MultiSelectList component since we may want to use it for other things in the future. Instead, we should be extending it by creating something like a UserMultiSelectList that wraps the MultiSelectList and adds any extra behaviour that we need
```javascript
export default class UserMultiSelectList extends React.PureComponent {
    componentDidMount() {
        loadStatusesFroProfilesList(this.props.options);
    }

    render() {
        return (
            <MultiSelectList
                {...this.props}
                optionRenderer={this.renderUser}
            />;
        );
    }
}
```